globals/plotTypes.json

```diff
@@ -0,0 +1,83 @@
+{
+  "m": {
+    "x": [
+      "m",
+      "sigma",
+      "radii",
+      "lnsigma",
+      "halo_bias",
+      "n_eff",
+      "dndm",
+      "ngtm",
+      "dndlnm",
+      "dndlog10m",
+      "rho_gtm",
+      "rho_ltm"
+    ],
+    "y": [
+      "central_occupation",
+      "cmz_relation",
+      "dndlnm",
+      "dndlog10m",
+      "dndm",
+      "fsigma",
+      "halo_bias",
+      "how_big",
+      "lnsigma",
+      "n_eff",
+      "ngtm",
+      "radii",
+      "rho_gtm",
+      "rho_ltm",
+      "satellite_occupation",
+      "sigma",
+      "total_occupation",
+      "tracer_cmz_relation",
+      "k_hm"
+    ]
+  },
+  "k": {
+    "x": ["k"],
+    "y": [
+      "delta_k",
+      "nonlinear_delta_k",
+      "nonlinear_power",
+      "power",
+      "transfer_function"
+    ]
+  },
+  "k_hm": {
+    "x": ["k_hm"],
+    "y": [
+      "power_1h_auto_matter",
+      "power_1h_auto_tracer",
+      "power_1h_cross_tracer_matter",
+      "power_1h_cs_auto_tracer",
+      "power_1h_ss_auto_tracer",
+      "power_2h_auto_matter",
+      "power_2h_auto_tracer",
+      "power_2h_cross_tracer_matter",
+      "power_auto_matter",
+      "power_auto_tracer",
+      "power_cross_tracer_matter"
+    ]
+  },
+  "r": {
+    "x": ["r"],
+    "y": [
+      "corr_1h_auto_matter",
+      "corr_1h_auto_tracer",
+      "corr_1h_cross_tracer_matter",
+      "corr_1h_cs_auto_tracer",
+      "corr_1h_ss_auto_tracer",
+      "corr_2h_auto_matter",
+      "corr_2h_auto_tracer",
+      "corr_2h_cross_tracer_matter",
+      "corr_auto_matter",
+      "corr_auto_tracer",
+      "corr_cross_tracer_matter",
+      "corr_linear_mm",
+      "sd_bias_correction"
+    ]
+  }
+}
```

Comment:
So just to be clear this bit isn't fully implemented yet? Each of these x values should correspond to everything in the y, ya?

Reply:
It is. All of the new ones that Steven requested are m labels. 
And yes, you have x vals should corrospond to everything in y.